lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -132,7 +132,7 @@ class LighthouseReportViewer {
   }
 
   /**
-   * @param {LH.ReportResult} json
```

Comment:
Really all the callers to this are either doing it with `any` or have casted `any`, so this is kind of a lie.

`json` should really have type `unknown` and we could have `_validateReportJson` verify that it's an `LH.Result` (as it's superficially doing already), but that can be a project for another day.

Should we add a TODO or file an issue?